frontend/src/pages/landing/Landing.tsx

```diff
@@ -43,56 +44,56 @@ import { LandingContext, getStatusMap } from "./Landing.loader";
 
 export type LandingKanbanEntry = {
   key: string;
-  onClick: () => void;
+  destructionList: DestructionList;
   disabled: boolean;
-  plannedDestructionDate: string | null;
-  processingStatus: ProcessingStatus;
   title: string;
   timeAgo: string;
-  assignees: React.ReactNode;
+  slotAssignees: React.ReactNode;
+  href?: string;
+  onClick?: () => void;
 };
 
 export const STATUSES: FieldSet<LandingKanbanEntry>[] = [
   [
     STATUS_MAPPING.new,
     {
-      fields: ["assignees"],
+      fields: ["slotAssignees"],
```

Comment:
What does the 'slotAssignee' mean?

I see that it is because it is the footer of the card where the assignees are shown, so it makes sense to give it a different name than "assignees". But I am not sure I understand the 'slot'

Reply:
I see :nerd_face: